Add private repo support (#12)
jazzsequence/dashboard-changelog: README.md

1. Download the zip file, unzip, and upload to your `/wp-content/plugins/` directory.
2. Activate Dashboard Changelog through the 'Plugins' menu in WordPress.
3. Go to your General Settings page, and add the repository you'd like to display updates from in <owner>/<repository-name> format (e.g. `jazzsequence/dashboard-changelog`).

> 4. If it is a private repository, also add a Github Personal Access token to allow the plugin to fetch data on your behalf. You can get a PAT in your [Github settings](https://github.com/settings/tokens), or by following this [official guide](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).

Reviewer: ```suggestion
4. For private repositories, add a GitHub Personal Access Token (PAT) to allow the plugin to fetch data on your behalf. You can get a PAT in your [GitHub settings](https://github.com/settings/tokens), or by following this [official guide](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
```

## `JSDC_REPOSITORY`
Global constant that can be used to hard-code the repository. If defined, the setting does not display in general settings. This constant can be defined in the `wp-config.php` file or elsewhere.

## `JSDC_PAT`

> Global constant that can be used to hard-code the PAT. If defined, the setting does not display in general settings. This constant can be defined in the `wp-config.php` file or elsewhere.

Reviewer: ```suggestion
Global constant that can be used to hard-code the Personal Access Token. If defined, the setting does not display in general settings. This constant can be defined in the `wp-config.php` file or elsewhere.
```

> **Note:** No validation are done at all on these strings, so make sure they're saved in the proper format.

Reviewer: ```suggestion
**Note:** There is no validation done on these strings, so make sure they're saved in the proper format.
```
